pages/quiz/how-are-you-feeling.tsx

```diff
@@ -248,10 +256,9 @@ export const HowAreYouFeelingPage = (): JSX.Element => {
           />
         </QuestionSection>
 
-        <Link href="/quiz/what-are-your-interests" passHref>
```

Comment:
We could keep this as a link and just choose the other URL

Reply:
This is true